themes/Default/Dark.qss

```diff
@@ -1085,4 +1085,33 @@ TalkingUI > QFrame {
   border-radius: 2px;
 }
 
+code {
+  background-color: #0d0620;
+}
+
+.codeblock > tr > * {
+  background-color: #0c0d27;
+  border: 1px solid #117d90;
+  border-right-color: #12727c;
+  border-bottom-color: #0a5861;
+  padding: 8px;
+}
+
+.blockquote > tr > * {
+  background-color: #1e1e1e;
+  color: #cbcac9;
+  border: 3px none #6e6e6e;
+  border-left-style: solid;
+  padding: 2px 5px;
+}
+
+.table {
+  border-width: 1px;
+  border-style: solid;
+  vertical-align: middle;
+}
+.table > tr > * {
```

Comment:
This missing newline here looks strange. Did you compile the ``.qss`` files according to the instructions on [this page](https://github.com/mumble-voip/mumble/tree/master/themes)?

If you did compile this, disregard my comment.